src/state/pools/fetchPools.ts

```diff
@@ -69,3 +71,29 @@ export const fetchPoolsTotalStaking = async () => {
     })),
   ]
 }
+
+export const fetchPoolStakingLimit = async (sousId: number): Promise<BigNumber> => {
+  try {
+    const sousContract = getSouschefV2Contract(sousId)
+    const stakingLimit = await sousContract.methods.poolLimitPerUser().call()
+    return new BigNumber(stakingLimit)
+  } catch (error) {
+    return BIG_ZERO
+  }
+}
```

Comment:
The `catch` here is important because for V1 pools

Reply:
this should be a comment in the code